go.mod

```diff
@@ -3,107 +3,118 @@ module github.com/gravitational/teleport
+	github.com/fsouza/fake-gcs-server v1.19.5
 	github.com/ghodss/yaml v1.0.0
-	github.com/gizak/termui v0.0.0-20190224181052-63c2a0d70943
-	github.com/gogo/protobuf v1.3.1
+	github.com/gizak/termui/v3 v3.1.0
+	github.com/gogo/protobuf v1.3.2
 	github.com/gokyle/hotp v0.0.0-20160218004637-c180d57d286b
-	github.com/golang/protobuf v1.4.2
+	github.com/golang/protobuf v1.4.3
 	github.com/google/btree v1.0.0
-	github.com/google/go-cmp v0.5.2
-	github.com/google/gops v0.3.1
-	github.com/gravitational/configure v0.0.0-20160909185025-1db4b84fe9db
+	github.com/google/go-cmp v0.5.4
+	github.com/google/gops v0.3.14
+	github.com/google/uuid v1.2.0 // indirect
+	github.com/gorilla/handlers v1.5.1 // indirect
+	github.com/gorilla/mux v1.8.0 // indirect
+	github.com/gorilla/websocket v1.4.2 // indirect
+	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20190130013101-742f2714c145+incompatible
-	github.com/gravitational/license v0.0.0-20180912170534-4f189e3bd6e3
+	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
 	github.com/gravitational/oxy v0.0.0-20200916204440-3eb06d921a1d
 	github.com/gravitational/reporting v0.0.0-20180907002058-ac7b85c75c4c
 	github.com/gravitational/roundtrip v1.0.0
 	github.com/gravitational/teleport/api v0.0.0
-	github.com/gravitational/trace v1.1.13
+	github.com/gravitational/trace v1.1.14
 	github.com/gravitational/ttlmap v0.0.0-20171116003245-91fd36b9004c
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/iovisor/gobpf v0.0.1
-	github.com/jackc/pgconn v1.7.0
-	github.com/jackc/pgproto3/v2 v2.0.7-0.20201117225705-88b6398594fc
-	github.com/johannesboyne/gofakes3 v0.0.0-20191228161223-9aee1c78a252
+	github.com/jackc/pgconn v1.8.0
+	github.com/jackc/pgproto3/v2 v2.0.7
+	github.com/johannesboyne/gofakes3 v0.0.0-20210217223559-02ffa763be97
 	github.com/jonboulle/clockwork v0.2.2
 	github.com/json-iterator/go v1.1.10
-	github.com/julienschmidt/httprouter v1.2.0
+	github.com/julienschmidt/httprouter v1.3.0
 	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0
 	github.com/kr/pty v1.1.8
-	github.com/kylelemons/godebug v0.0.0-20160406211939-eadb3ce320cb
-	github.com/mailgun/lemma v0.0.0-20160211003854-e8b0cd607f58
+	github.com/kylelemons/godebug v1.1.0
+	github.com/mailgun/lemma v0.0.0-20170619173223-4214099fb348
 	github.com/mailgun/metrics v0.0.0-20150124003306-2b3c4565aafd // indirect
-	github.com/mailgun/minheap v0.0.0-20131208021033-7c28d80e2ada // indirect
-	github.com/mailgun/timetools v0.0.0-20141028012446-7e6055773c51
-	github.com/mailgun/ttlmap v0.0.0-20150816203249-16b258d86efc
-	github.com/mattn/go-runewidth v0.0.4 // indirect
+	github.com/mailgun/minheap v0.0.0-20170619185613-3dbe6c6bf55f // indirect
+	github.com/mailgun/timetools v0.0.0-20170619190023-f3a7b8ffff47
+	github.com/mailgun/ttlmap v0.0.0-20170619185759-c1c17f74874f
```

Comment:
Isn't this master from Mailgun, I thought we were at `91fd36b9004c1ee5a778739752ea1aba5638c03a` from `github.com/gravitational/ttlmap`?

Reply:
`github.com/mailgun/ttlmap` is used in a few places instead of our fork.
let me update them, should be low-risk

Reply:
Ugh, the mailgun stuff is tightly woven into `lib/limiter` and `vendor/github.com/gravitational/oxy/ratelimit/tokenlimiter.go` so we can't get rid of it from our deps entirely.